### PR TITLE
docs: add keywords to inverted index page

### PR DESCRIPTION
**Pull request by DanRoscigno**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/engines/table-engines/mergetree-family/invertedindexes.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 sidebar_label:  Inverted Indexes
+description: Quickly find search terms in text.
+keywords: [full-text search, text search]
 ---
 
 # Inverted indexes [experimental]
 
-Inverted indexes are an experimental type of [secondary indexes](mergetree.md#available-types-of-indices) which provide fast text search
-capabilities for [String](../../../sql-reference/data-types/string.md) or [FixedString](../../../sql-reference/data-types/fixedstring.md)
+Inverted indexes are an experimental type of [secondary indexes](/docs/en/engines/table-engines/mergetree-family/mergetree.mdi/#available-types-of-indices) which provide fast text search
```
Comment (rschu1ze) [inline docs/en/engines/table-engines/mergetree-family/invertedindexes.md:10]: Is that really "mergetree.mdi"?

Comment (DanRoscigno) [reply docs/en/engines/table-engines/mergetree-family/invertedindexes.md:10]: thanks!

Comment (DanRoscigno) [inline docs/en/engines/table-engines/mergetree-family/invertedindexes.md:10]:
```suggestion
Inverted indexes are an experimental type of [secondary indexes](/docs/en/engines/table-engines/mergetree-family/mergetree.md/#available-types-of-indices) which provide fast text search
```

**Review by rschu1ze — APPROVED** [review]

**Comment by DanRoscigno** [comment]
Thanks Robert!

**Pull request merged**